.travis.yml

```diff
@@ -64,11 +66,12 @@ matrix:
         - redis-server
         - postgresql
       before_install:
-        - cd $TRAVIS_BUILD_DIR/src/$COMPONENT && bundle config build.nokogiri --use-system-libraries
+        - cd src/supermarket/engines/fieri
+        - bundle config build.nokogiri --use-system-libraries
       before_script:
-        - cd $TRAVIS_BUILD_DIR/src/$COMPONENT && bundle exec rake $COMPONENT:travis:before_script
+        - bundle exec rake fieri:travis:before_script
       script:
-        - cd $TRAVIS_BUILD_DIR/src/$COMPONENT && bundle exec rake $COMPONENT:travis:script
+        - bundle exec rake fieri:travis:script
```

Comment:
The `cd src/supermarket/engines/fieri` in `before_install` was sufficient for getting into the working directory needed for testing. This matches the style of the other test jobs added to Travis for the pipeline.